docs/xml/metainfo-component.xml

```diff
@@ -834,6 +834,89 @@
 				</listitem>
 			</varlistentry>
 
+			<varlistentry id="tag-requires-recommends-display-width-height">
+				<term>&lt;display_length/&gt;</term>
+				<listitem>
+				<para>
+					Set a relation to the display length defined as an integer value in either <emphasis>millimeters</emphasis> or <emphasis>logical pixels</emphasis>
```

Comment:
I'm still not sure what the potential use cases for millimeters are, so unless there's something else I'm missing I'd stick to having logical pixels only.

Reply:
It's mimicking the existing Qt and GTK+ APIs as well as future-proofing the design (some adaptive apps may care more about the form factor than the drawing distance).

Reply:
Which GTK APIs? GdkMonitor etc? Not sure what an app would use them for tbh, other than misusing them like in https://github.com/stdmedoth/simplenav/blob/dcfe229abcb1a6e31321407ae8855b8ad4c74fc7/src/init.c#L18-L20